src/Utility.js

```diff
@@ -30,10 +32,11 @@ export function trim (value) {
 /**
  * @param {string} value
  * @param {RegExp} pattern
- * @return {string?}
+ * @return {string}
```

Comment:
Never null?

Reply:
notice how you have changed what this function does, previously `value` was reassigned as the code was written to be as terse as possible, so it was actually possible that we could have returned `null` from it as `.match` call might return it. Right now you are returning the input value and this is not an equivalent